### DISCUSSION TITLE
xDrip multiple activity/setting profiles

### DISCUSSION THREAD

**Discussion by spanosp** [NightscoutFoundation/xDrip#2407] [Ideas]
Would it be possible to have multiple "activity profiles" where each profile has different settings?

In my mind, it would be  great if I could switch from one activity profile to another and easily switch from one set of settings to another.
For example, during my normal daily routine, I dont use "speach readings" while when I go running I would like to have these on a regular basis (eg every 5 min).

**Comment by Navid200** [comment]
You can set up xDrip for one of your conditions, as you call it profile.
Then, you can save the settings as a file or a QR code.  That's up to you.

Now, repeat that for another profile.

Name the settings, or QR code images, accordingly so that you can easily identify them later.


You can load each when needed.

**Reply by spanosp** [reply]
Hi @Navid200, indeed you could do that but I beleive its not so convenient for swapping profiles back and forth before/after an activity.
